### DISCUSSION TITLE
QZ-qdomyos-zwift

### DISCUSSION THREAD

**Discussion by martinwprior** [cagnulein/qdomyos-zwift#1165] [Q&A]
I have installed this on my rasberry pi 4 and I get a really nice display and my cadence sensor and Heartrate monitor are showing up. My question is that when I used it on the android phone,, the phone acts as an FTMS server (?) and it can be picked up by Zwift etc. 

I cannot see this from the Raspberry Pi? Should I? Do I need to setup something or install something?

I see you have a Netgear device in one of your posts. I do have a Magene ANT+ usbwhich shows as id Bus 001 Device 011: ID 0fcf:1008 Dynastream Innovations, Inc. ANTUSB2 Stick

**Comment by cagnulein** [comment]
yes raspi is doing the same. please share a debug log

**Comment by martinwprior** [comment]
call me dumb but where does the debug file go. Debug is switched on in the
app. I used find to search for recent files but cannot find any.

**Reply by cagnulein** [reply]
on raspberry are in the same folder as the binary
